### DISCUSSION TITLE
MSFS blocking at 50%

### DISCUSSION THREAD

**Discussion by cro24** [derekhe/msfs2020-google-map#173] [Q&A]
I can only open MSFS once in 5 or 6; because blocking at about 50%. Forced to kill the process by the task manager.
I launch the application before launching 2020, I position the two windows on a second screen, and stop the application, before killing 2020. I don't have a popup window for the certificate. My hosts file is empty
Thanks for your help because I love your job;
Thank you,
Edit: 31 december: Alleluia everything started after a few tries, I didn't do anything in particular. Weird ( After cleaning the rolling cache)
Edit 01 january: And again and again, too bad because this mod is terrible
Edit 02 january: i have deleting le rolling cache, and i think that is a solution, i hope
P.S. sorry for my english (google translation)

**Comment by ricdpm** [comment]
Have you checked my workaround in flightsim.to mod page?
My name is **rdpm**

**Reply by cro24** [reply]
> 

I read your solution correctly, but I think removing the rolling cache, and one more. While waiting for the new version, I remain on this process.
